### PR TITLE
TRAN-135: Filtering based on shipment cost added

### PR DESCRIPTION
**Pull request by dborowiecki**
Addtional changes cover shipment order improvement, separate endpoint for adding removing or replacing orders in scope of one shipment. 
TICKET: [TRAN-135](https://sd-transit.atlassian.net/browse/TRAN-135)

### CODE REVIEW

File: transit/models/shipment.py [hunk 1]
```diff
@@ -1,11 +1,67 @@
...
+
+class ShipmentDetailsQueryset(models.QuerySet):
+
+    def transporter_shipments(self, transporter: Transporter):
+        return self.filter(transporter_details__transporter=transporter).distinct()
+
+    def vehicle_shipments(self, vehicle: TransporterDetails = None, vehicles: Collection[TransporterDetails] = None):
+        """
+        Filter ShipmentDetails for shipments relevant for given vehicle or vehicles list.
+        Only on of `vehicle` and `vehicles` arugments can be provided. If both are provided ValueError is raised.
```
Comment (sniedzielski) [inline transit/models/shipment.py:21]: typo: 'arugments' -> 'arguments'

Comment (sniedzielski) [inline transit/models/shipment.py]: typo: 'arugments' -> 'arguments'

**Review by sniedzielski — COMMENTED** [review]
looks good, there is one minor typo

File: transit/models/shipment.py (hunk 1 above)
Comment (dborowiecki) [reply transit/models/shipment.py]: Fixed

**Review by sniedzielski — APPROVED** [review]

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**